lib/shared/drive-constraints.js

```diff
@@ -303,7 +304,7 @@ exports.COMPATIBILITY_STATUS_MESSAGES = {
    * @description
    * The drive is too small for the image.
    */
-  TOO_SMALL: 'Too Small For Image',
+  TOO_SMALL: 'Insufficient space, additional XX required',
```

Comment:
Maybe we should make these into template arrow functions? We can keep it as-is now and open a new PR for that, though.

Reply:
I've now moved the compatibility messages from `drive-constraints` to `shared/messages` and made them functions